app/views/mortgage_calculator/affordabilities/_form_step1.html.erb

```diff
@@ -106,7 +132,7 @@
                               "name" => "affordability[two_applicants]",
                               "affects-height" => "change",
                               :checked => @affordability.two_applicants? %>
-              <%= I18n.t("affordability.titles.second_applicant") %> <span class="visually-hidden">(displays new content below)</span>
+              <span class="label--inline"><%= I18n.t("affordability.titles.checkbox_label") %></span>
```

Comment:
Can we use single-quotes on the ruby erb part, please?

Reply:
Hi @tomas-stefano I updated for single quotes in the erb file.